pom.xml

```diff
@@ -941,11 +1000,22 @@
         </plugin>
 
         <plugin>
+          <!-- Set up Javadoc configuration for javadoc run via in the Maven build.
+               Beam only uses the plugin to create the individual javadoc attached artifacts
+               used for IDEs. The combined javadoc for the website is built in the
+               aggregate-javadoc directory. Rather than duplicate a raft of configuration
+               between the ant.xml there and here, we leave things simple here. -->
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
+          <configuration>
+            <additionalparam>${beam.javadoc_opts}</additionalparam>
+            <windowtitle>Apache Beam SDK for Java, version ${project.version} API</windowtitle>
+            <doctitle>Apache Beam SDK for Java, version ${project.version}</doctitle>
+            <use>false</use>
```

Comment:
Use adds extra pages that I do not think are useful in the javadoc artifacts:  [use](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#use).

quiet: 'Shuts off non-error and non-warning messages, leaving only the warnings and errors appear, making them easier to view. ' 

which makes sense to me for building the attached javadoc artifacts.